docker/hive/Dockerfile

```diff
@@ -134,7 +134,6 @@ ENV KAKAROT_RPC_URL=0.0.0.0:8545
 ENV STARKNET_NETWORK=http://localhost:5050
 ENV RUST_LOG=kakarot_rpc=info
 ENV MAX_FELTS_IN_CALLDATA=30000
-ENV RETRY_TX_INTERVAL=10
```

Comment:
ah sorry, what I meant is that we don't need it in the `ALLOW_ENV_FROM_ENV` but I think we need to have it in the env otherwise RPC won't start no?

Reply:
Yes correct, but I was thinking that the variables in `ALLOW_ENV_FROM_ENV` are the ones that we authorized as env variables, that's why I was confused no?